docs/config/stack.md

```diff
@@ -0,0 +1,22 @@
+## Stack Configuration
+
+
+### Function Config
+
+Functions can be configured with memory and timeout requirements per function, a default can also be set for all functions in a stack that do not have their own configuration.
```

Comment:
It would probably be worth specifying what the default is when this configuration is not set

Reply:
Have added a sub section on default values.